hw/ip_templates/clkmgr/data/clkmgr.tpldesc.hjson

```diff
@@ -86,5 +86,23 @@
       type: "int"
       default: "2"
     }
+    {
+      name: "with_alert_handler"
+      desc: "Generate outputs for a clkmgr that would connect to an alert handler"
+      type: "bool"
+      default: "1"
+    }
+    {
+      name: "pwrmgr_vlnv_prefix"
```

Comment:
Another side-effect of the hopefully-to-be-replaced instance_vlnv implementation.

Reply:
They might change some, but I don't think we can remove these template variables altogether. At some point, we need to allow multiple specializations of clkmgr / pwrmgr / rstmgr in one topgen run.